lib/repo/git_repository.rb

```diff
@@ -30,15 +30,15 @@ class GitRepository < Repository::AbstractRepository
     # created using GitRespository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
+      if IS_REPOSITORY_ADMIN.nil?
```

Comment:
This has come up in another context.  There seem to be three ways to get this info:  Environment variable IS_REPOSITORY_ADMIN, Repository.conf[:IS_REPOSITORY_ADMIN], and MarkUsConfigurator.markus_config_repository_admin

I'd prefer to get rid of Repository.conf altogether because there is an issue with when it is created, and it really does seem like global configuration.  But, which of the other two is "better"?